README.md

```diff
@@ -76,40 +76,82 @@ Once run, the service can be verified/accessed at [http://localhost:8888/fester/
 
 ## Debugging with Eclipse IDE
 
-You can run a development instance of Fester with debugging turned on by typing the following within the project root:
+There are two ways to debug Fester:
 
-    mvn -Pdebug test
-Development instances are configured to accept remote debugger connections on port `5555`.
+- **Debugging the tests.** This enables stepping through test suite code as well as the application code as runs according to the test suite.
+- **Debugging a running instance.** This enables stepping through the application code as it runs according to a developer's interaction with the HTTP API.
```

Comment:
Alternative wording(?)
> This enables the developer to step through the application code as he or she interacts with the HTTP API.

Or, could be "as they interact with the HTTP API" as a gender neutral option.